### ISSUE TITLE
Shoes "builtins" should be defined on Kernel

### ISSUE THREAD

**Issue by noahgibbs** [scarpe-team/scarpe#300]
In Shoes4, this was the list of builtins that should be available everywhere:

```
    alert
    confirm / confirm?
    info
    debug
    warn
    error
    ask_open_file
    ask_save_file
    ask_open_folder
    ask_save_folder
    ask
    ask_color
    font
```

Shoes3 includes all the same ones, plus a few more:

```
rgb
gray
rgb
gradient
pattern
```

**Comment by noahgibbs** [comment]
The basic mechanics are fixed by #410, though that doesn't implement all the functionality.

**Comment by noahgibbs** [comment]
Don't think we need to leave this bug open to keep adding APIs, though.
